libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -557,15 +557,72 @@ - (void)beginUserAgentFlow {
         [approvalUrl appendString:scopeString];
     }
     
+    // JWT Flow
+    if (self.credentials.jwt && self.credentials.instanceUrl) {
+        [self swapJWTWithcompletionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
+            bool swapOK = NO;
+            if (!error) {
+                NSError *jsonError = nil;
+                id json = nil;
+                
+                json = [NSJSONSerialization JSONObjectWithData:data options:0 error:&jsonError];
```

Comment:
Ah, throwbacks to when OAuth lived at a lower layer than SalesforceSDKCore.  We can save that for another time, as it's consistent (as you point out) and not related to this PR.  My $0.02, anyway.